src/tests/zf_unit/zfattrs.c

```diff
@@ -16,6 +16,7 @@ struct attr_value test_attrs[] = {
   { "log_level", 0x7777 },
   { "log_format", 15 },
   { "log_to_kmsg", 1 },
+  { "log_file", 0 },
```

Comment:
For everything else, these are attribute values used in the test.  I think this could just be set to a single filename rather than using multiple files.  I'm not sure what additional value there is in testing with different files on different iterations rather than same file since there are no output checks here.

I've put further review comments below independent of above comment

Reply:
Sounds reasonable, but wanted to test the case of alternating between stderr and different files as per zf_attrs.c test. Moreover, it's what's done in the test, repeated tests randomly picking different attribute settings one of which is log_file which I believe it deserves to be independent on each iteration.

Reply:
ok.  I don't think it changes the effectiveness of the test either way.